subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/dsl/DefaultRepositoryHandler.java

```diff
@@ -124,7 +124,7 @@ public MavenArtifactRepository jcenter(Action<? super MavenArtifactRepository> a
 
     private void deprecateJCenter(String method, String replacement) {
         DeprecationLogger.deprecateMethod(RepositoryHandler.class, method)
-            .withAdvice("JCenter will soon be shut down. Use " + replacement + " instead.")
+            .withAdvice("JCenter shut down was announced in February 2021. Use " + replacement + " instead.")
```

Comment:
Maybe "The JCenter shutdown was announced in February 2021." or "JFrog announced JCenter's shutdown in February 2021"